tests/tts_dashscope_cosyvoice_test.py

```
@@ -0,0 +1,426 @@
+        mock_modules[
+            "dashscope.audio.tts_v2"
+        ].SpeechSynthesizer.return_value = mock_synthesizer_instance
+
+        with patch.dict("sys.modules", mock_modules):
+            with patch(
+                "agentscope.tts._dashscope_cosyvoice_realtime_tts_model"
+                ".get_cosyvoice_callback_class",
+            ) as mock_get_callback:
+                mock_callback_class = MagicMock()
+                mock_callback = self._create_mock_callback()
+
+                async def mock_generator() -> AsyncGenerator[
+                    TTSResponse,
+                    None,
+                ]:
+                    yield TTSResponse(
+                        content=AudioBlock(
+                            type="audio",
+                            source=Base64Source(
+                                type="base64",
+                                data=self.mock_audio_base64,
+                                media_type="audio/pcm;rate=24000",
+                            ),
+                        ),
+                        is_last=False,
+                    )
+                    yield TTSResponse(
+                        content=AudioBlock(
+                            type="audio",
+                            source=Base64Source(
+                                type="base64",
+                                data=self.mock_audio_base64,
+                                media_type="audio/pcm;rate=24000",
+                            ),
+                        ),
+                        is_last=True,
+                    )
+
+                mock_callback.get_audio_chunk = Mock(
+                    return_value=mock_generator(),
+                )
+                mock_callback_class.return_value = mock_callback
+                mock_get_callback.return_value = mock_callback_class
+
+                from agentscope.tts._dashscope_cosyvoice_realtime_tts_model import (  # noqa: E501
```

We should avoid such import rather than ignore this error